appendix-petsciicodes.tex

```diff
@@ -231,102 +233,105 @@ \section{Control codes}
+\specialkey{CTRL} + \megakey{T} &
+Backspace the character immediately to the left and to shift all rightmost characters one position to the left. This is equivalent to the \specialkey{INST DEL} key.\\
 \hline
-\megakey{CTRL} + \megakey{M} &
-Performs a carriage return, the same function as the \specialkey{RETURN} key.\\
+\specialkey{CTRL} + \megakey{M} &
+Performs a carriage return, equivalent to the \specialkey{RETURN} key.\\
 
   \hhline{==}
   \multicolumn{2}{l}{\textbf{Word movement}} \\
   \hhline{==}
 
-\megakey{CTRL} + \megakey{U} &
-Moves cursor back to the start of the previous word, or unbroken string of characters. If there are no characters between the current cursor position and the start of the line, the cursor will move to the first column of the current line.\\
+\specialkey{CTRL} + \megakey{U} &
+Moves the cursor back to the start of the previous word. If there are no words
+between the current cursor position and the start of the line, the cursor will move to the first column of the current line.\\
 \hline
-\megakey{CTRL} + \megakey{W} &
-Advances cursor forward to the start of the next word, or unbroken string of characters. If there are no characters between the current cursor position and the end of the line, the cursor will move to the first column of the next line.\\
+\specialkey{CTRL} + \megakey{W} &
+Advances the cursor forward to the start of the next word. If there are no words between the cursor and the end of the line,
+the cursor will move to the first column of the next line.\\
 
   \hhline{==}
   \multicolumn{2}{l}{\textbf{Scrolling}} \\
   \hhline{==}
 
-\megakey{CTRL} + \megakey{P} &
-Scroll BASIC listing down one line, equivalent to \megakey{F9} key.\\
+\specialkey{CTRL} + \megakey{P} &
+Scroll BASIC listing down one line. Equivalent to the \megakey{F9} key.\\
 \hline
-\megakey{CTRL} + \megakey{V} &
-Scroll BASIC listing up one line, equivalent to \megakey{F11} key.\\
+\specialkey{CTRL} + \megakey{V} &
+Scroll BASIC listing up one line. Equivalent to the \megakey{F11} key.\\
 \hline
-\megakey{CTRL} + \megakey{S} &
-Equivalent to \specialkey{NO\\SCROLL} key.\\
+\specialkey{CTRL} + \megakey{S} &
+Equivalent to the \specialkey{NO\\SCROLL} key.\\
 
   \hhline{==}
   \multicolumn{2}{l}{\textbf{Formatting}} \\
   \hhline{==}
 
-\megakey{CTRL} + \megakey{B} &
-Turns on underline text mode. Turn off underline mode by pressing \megakey{ESC} then \megakey{O}.\\
+\specialkey{CTRL} + \megakey{B} &
+Enables underline text mode. You can disable underline mode by pressing \specialkey{ESC}, then \megakey{O}.\\
 \hline
-\megakey{CTRL} + \megakey{O} &
-Turns on flashing text mode. Turn off flashing mode by pressing \megakey{ESC} then \megakey{O}.\\
+\specialkey{CTRL} + \megakey{O} &
+Enables flashing text mode. YOu can disable flashing mode by pressing \specialkey{ESC}, then \megakey{O}.\\

```

Comment:
Spotted a typo in 'YOu' instead of 'you'.

Also happy for you to switch the " 'ESC, then 'O' " wording to your briefer suggestion of 'ESC O', if you think it's appropriate.